app/api/__init__.py

```diff
@@ -133,3 +138,9 @@
 api.route(EventInvoiceDetail, 'event_invoice_detail', '/event-invoices/<int:id>')
 api.route(EventInvoiceRelationship, 'event_invoice_user', '/event-invoices/<int:id>/relationships/user')
 api.route(EventInvoiceRelationship, 'event_invoice_event', '/event-invoices/<int:id>/relationships/event')
+
+# speakers
+api.route(SpeakerList, 'speaker_list', '/speakers', '/events/<int:event_id>/speakers')
```

Comment:
`/speakers` is not required I think, speaker should exist only as relationship to an event